contracts/Remittance.sol

```diff
@@ -1,16 +1,129 @@
 //SPDX-License-Identifier: MIT
 pragma solidity >= 0.6.0 < 0.7.0;
 
-import "../node_modules/@openzeppelin/contracts/access/Ownable.sol";
-import "../node_modules/@openzeppelin/contracts/utils/Pausable.sol";
+import "./access/Ownable.sol";
+import "./utils/Pausable.sol";
 
-import {SafeMath} from "../node_modules/@openzeppelin/contracts/math/SafeMath.sol";
+import {SafeMath} from "./math/SafeMath.sol";
 
 contract Remittance is Ownable, Pausable {
 
     using SafeMath for uint;
 
+    mapping(address => uint) public balances;
 
+    mapping(bytes32 => RemittanceContract) public contracts;
+
+    struct RemittanceContract {
+        address funder;
+        address broker;
+        address recipient;
+        uint amount;
+        bool fundsReleased;
+    }
+
+    event Deposit(
+        address indexed depositor,
+        uint amount
+    );
+
+    event RemittanceContractCreated(
+        bytes32 hash,
```

Comment:
I think `hash` ought to be `indexed` as it is the guiding element of the lifecycle, and a question will be asked as to "what happened to my hash"?